google-cloud-bom/pom.xml

```diff
@@ -354,6 +354,21 @@
         <artifactId>google-cloud-core-http</artifactId>
         <version>1.69.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-core-http:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-datacatalog</artifactId>
+        <version>0.87.1-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-datacatalog:current} -->
```

Comment:
shouldn't the version be 0.0.1-alpha-SNAPSHOT?

Reply:
We have been using a consistent version across all artifacts that are pre-0.  I'm not sure what we ought to do going forward.